### PR TITLE
Beth/sponsor footer

### PR DESCRIPTION
**Pull request by bhg**

### CODE REVIEW

**Review by marlonkeating — CHANGES_REQUESTED** [review]
You'll need to include the migration file for this, which you generate with:
python manage.py makemigrations

**Comment by bhg** [comment]
> You'll need to include the migration file for this, which you generate with:
> python manage.py makemigrations

Added this now.

File: common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx [hunk 1]
```diff
@@ -118,6 +119,7 @@ class AboutEventDisplay extends React.PureComponent<Props, State> {
         {!_.isEmpty(event.event_legacy_organization) && (
           <ProfileProjectSearch viewOnly={this.props.viewOnly} />
         )}
+        <MainFooter key="main_footer" forceShow={event.show_headers}/>
```
Comment (marlonkeating) [inline common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx:122]: The MainFooter component was designed to be full-width, and inside the "AboutEvent-root container" div puts it within a smaller frame.  I would either put it in the AboutEventController component, or just outside the div in this component.

**Review by marlonkeating — CHANGES_REQUESTED** [review]

File: common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx (hunk 1 above)
Comment (bhg) [reply common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx:124]: Done and tested.

File: common/components/chrome/MainFooter.jsx [hunk 2]
```diff
@@ -18,9 +18,13 @@ const sectionsToShowFooter: $ReadOnlyArray<string> = [
...
 ];
 
-class MainFooter extends React.Component<{||}> {
-  constructor(): void {
-    super();
+type Props = {|
+  forceShow: boolean,
+|};
+
+class MainFooter extends React.Component<{||}, Props> {
```
Comment (marlonkeating) [inline common/components/chrome/MainFooter.jsx:25]:
```suggestion
class MainFooter extends React.Component<Props> {
```

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**